Need to link against the C math library (-lm) at least for uclibc.

Index: fcgi-2.4.0/cgi-fcgi/Makefile.am
===================================================================
--- fcgi-2.4.0.orig/cgi-fcgi/Makefile.am	2008-07-09 20:54:56.000000000 +0200
+++ fcgi-2.4.0/cgi-fcgi/Makefile.am	2008-07-09 20:55:12.000000000 +0200
@@ -11,5 +11,5 @@
 LIBDIR      = ../libfcgi
 LIBFCGI     = $(LIBDIR)/libfcgi.la
 
-LDADD = $(LIBFCGI)
+LDADD = $(LIBFCGI) -lm
 cgi_fcgi_SOURCES = $(INCLUDE_FILES) cgi-fcgi.c
Index: fcgi-2.4.0/examples/Makefile.am
===================================================================
--- fcgi-2.4.0.orig/examples/Makefile.am	2008-07-09 20:54:56.000000000 +0200
+++ fcgi-2.4.0/examples/Makefile.am	2008-07-09 20:55:11.000000000 +0200
@@ -21,7 +21,7 @@
 LIBDIR      = ../libfcgi
 LIBFCGI     = $(LIBDIR)/libfcgi.la
 
-LDADD = $(LIBFCGI)
+LDADD = $(LIBFCGI) -lm
 
 echo_SOURCES       = $(INCLUDE_FILES) echo.c
 echo_x_SOURCES     = $(INCLUDE_FILES) echo-x.c
